resources/ThePath.xml

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Rename the xml file to something more meaningful than ThePath.xml (e.g. SchoolData, SchoolStructure, etc.)